mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/MailboxElasticSearchConstants.java

```diff
@@ -28,4 +28,6 @@
     AliasName DEFAULT_MAILBOX_READ_ALIAS = new AliasName("mailboxReadAlias");
     IndexName DEFAULT_MAILBOX_INDEX = new IndexName("mailbox_v1");
     TypeName MESSAGE_TYPE = new TypeName("message");
+
+    String MAILBOX = "mailbox";
```

Comment:
what is this?

Reply:
name for injections. Maybe MAILBOX_INJECT_NAME is better...